features/cli.feature

```diff
@@ -3,6 +3,21 @@ Feature: Command line interface
   As a person who wants to run features
   I want to run Cucumber on the command line
 
+  Scenario: run with no features directory
+    When I run cucumber.js with `features`
```

Comment:
Thoughts on making this line
```
When I run cucumber.js
```
I doubt the user to pass in the features argument with no features folder created and features is the default argument if none is passed anyway?